projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -114,11 +114,21 @@ export class NavigationUIComponent implements OnInit, OnDestroy {
     }
   }
 
-  toggleOpen(event: UIEvent): void {
+  toggleOpen(event: UIEvent, navNode: NavigationNode): void {
+    const node = <HTMLElement>event.currentTarget;
+
+    if (typeof navNode.url === 'string') {
+      if (window.location.href.includes(navNode.url) && !navNode.children) {
```

Comment:
the second condition seems as not necessary for fix
```suggestion
      if (window.location.href.includes(navNode.url)) {
```

Reply:
Hey @Pucek9 I tried it with your suggestion but then without the second condition there is bug. To see what I mean, remove the second condition and load the app. Then in mobile view, open the menu, click on 'DIGITAL CAMERAS' then click on 'Shop all Digital Cameras>'. If you try to navigate again to 'DIGITAL CAMERAS', it is not possible and subcategories of Digital Cameras become inaccessible

Reply:
@BenjaminEmiliani You're right. So if there are no contraindications, let's simplify only.
```suggestion
      if (typeof navNode.url === 'string' && window.location.href.includes(navNode.url) && !navNode.children) {
```